concent_api/core/message_handlers.py

```diff
@@ -389,6 +389,9 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
         task_to_compute,
     )
 
+    if Subtask.objects.filter(subtask_id=task_to_compute.compute_task_def['subtask_id']).exists():  # pylint: disable=no-member
+        Subtask.objects.select_for_update().get(subtask_id=task_to_compute.compute_task_def['subtask_id'])  # pylint: disable=no-member
+
     if Subtask.objects.filter(  # pylint: disable=no-member
         subtask_id = task_to_compute.compute_task_def['subtask_id'],
         state      = Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name,  # pylint: disable=no-member
```

Comment:
You have already performed a query above and locked the subtask. This code can now just use the task you fetched instead of fetching it again. You could replace it with 
``` python
if subtask.state == Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name:
```

BTW, don't we have a custom field that lets you drop the `.name` here?

Reply:
right, after small refactor changed. And there is possibility to drop `.name` by comparing `enums` instead of `strings`

Reply:
Yeah. I thought we refactored it in other places already. After you're done with `select_for_update()` you could make a commit that removes those unnecessary `.name`s. We want to use enums in comparisons if possible.